src/interpreter.erl

```diff
@@ -43,49 +50,62 @@ move_index_on_tape({Index, Tape, right}) ->
             {RightIndex, Tape}
     end.
 
-print_tape_and_head_on_tape(IndexOnTape, Tape) ->
+print_tape_and_head_on_tape(IndexOnTape, Tape, CurrentState) ->
     io:format("Tape: ["),
-    print_tape_and_head_on_tape(IndexOnTape, Tape, 1),
-    io:format("]~n").
+    print_tape_and_head_on_tape(IndexOnTape, Tape, 1, CurrentState),
+    io:format("] STATE=~p~n", [CurrentState]).
 
-print_tape_and_head_on_tape(IndexOnTape, Tape, CurrentIndexOnTape) ->
+print_tape_and_head_on_tape(IndexOnTape, Tape, CurrentIndexOnTape, CurrentState) ->
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Does this function really need to take `CurrentState` as parameter?